File: Gerontocracy.Core/BusinessObjects/Board/ThreadOverview.cs
using System;

namespace Gerontocracy.Core.BusinessObjects.Board
{
    public class ThreadOverview
    {
        public long Id { get; set; }
        public string Titel { get; set; }
        public long? VorfallId { get; set; }
        public string VorfallTitel { get; set; }
        public long? PolitikerId { get; set; }
        public string PolitikerName { get; set; }
        public DateTime CreatedOn { get; set; }
        public long? UserId { get; set; }
        public string UserName { get; set; }
        public long NumPosts { get; set; }
        public bool Generated { get; set; }
    }
}

File: Gerontocracy.Web/src/gerontocracy/modules/board/models/thread-overview.ts
export interface ThreadOverview {
    id: number;
    titel: string;
    vorfallId: number;
    vorfallTitel: string;
    politikerId: number;
    politikerName: string;
    createdOn: Date;
    userId: number;
    userName: string;
    numPosts: number;
    generated: boolean;
}
